packages/mui-material/src/Badge/Badge.js

```diff
@@ -6,7 +6,7 @@ import { usePreviousProps } from '@mui/utils';
 import { unstable_composeClasses as composeClasses } from '@mui/base/composeClasses';
 import { useBadge } from '@mui/base/useBadge';
 import { useSlotProps } from '@mui/base';
-import styled from '../styles/styled';
+import { styled } from '../zero-styled';
```

Comment:
Added alternate way to specify zero-runtime readiness without the need of a separate package.
The other approach seemed unnecessarily complex and required a side-effect file import.